daily-challenges/2024-09/14/thao.ha93.spec.ts

```diff
@@ -0,0 +1,49 @@
+function calculateBMI(weight: number, height: number) {
+    let BMI = 0;
+    if (weight <= 0 && height <= 0) {
+        throw new Error('Please recheck height or weight');
+    }
+    BMI = weight / (height * height);
+    console.log(`Kết quả BMI: ${BMI}`);
+
+    if (BMI < 18.5) {
+        console.log('Phân loại: "Gầy"');
+    }
+    else if (BMI >= 18.5 && BMI < 24.9) {
+        console.log('Phân loại: "Bình thường"');
+    }
+    else if (BMI >= 25 && BMI < 29.9) {
+        console.log('Phân loại: "Thừa cân"');
+    }
+    else {
+        console.log('Phân loại: "Béo phì"');
+    }
+}
+let weight = 53, height = 1.5;
+calculateBMI(weight, height);
+
+//Playwright
```

Comment:
Các dòng comment sẽ đẹp hơn nếu có khoảng trống giữa // và text
```suggestion
// Playwright
```